### DISCUSSION TITLE
数学大模型项目--不要葱姜蒜队

### DISCUSSION THREAD

**Discussion by KMnO4-zx** [InternLM/Tutorial#163] [project]
## 项目目的 && 意义

**_项目的主要目的是为了验证大模型是否能够在大量的已有数学知识训练下完成未知高等数学题目的解答~
主要目的是学习~  不是竞争！_**

**_高等数学的习题集天然的就形成大模型 SFT 训练的数据集，本实验基于高等数据习题及习题解析来制作数据集。基座模型暂定为Qwen-1.8B、InternLM-7B、InternLM-20B_**

## 项目成员

- _不要葱姜蒜_
- _小L非常困_
- _A=X+Y+Z_
- _散步_
- _Logan_

## 项目文档

[数学大模型项目文档](https://vj6fpcxa05.feishu.cn/docx/FMbFdfiAuoOrRNxQzCMcYCebnLf)

![image](https://github.com/InternLM/tutorial/assets/77671993/c967d72a-333c-45eb-a605-01c8b72279f7)

**Comment by blurmemo** [comment]
佬可以带带吗

**Reply by KMnO4-zx** [reply]
已经人满了哟~

**Comment by Tianhang-Lu** [comment]
可以一起组队吗，学习态度认真

**Reply by KMnO4-zx** [reply]
已经人满了哟~
